packages/docs/docs/install/pikapods.md

```diff
@@ -4,85 +4,88 @@ title: 'PikaPods'
 
 
 [PikaPods](https://www.pikapods.com/) offers one click "instant open source app hosting", allowing
-you to run Actual for less than $ 1.50 per month (as of July 2024).
+you to run Actual for about $ 1.50 per month (as of November 2025).
 
 Using PikaPods is also a simple way to support the development of Actual Budget, as PikaPods will
 share some of its revenues with Actual for customers using their Actual Budget pods.
 
-You get a $ 5.00 credit when you sign up, which means that you should be able to run Actual for
-over 3 months before your credit runs out.
+You get a $ 5.00 credit when you sign up, which means that you should be able to run Actual for 3 months before your credit runs out.
 
-For web clients, PikaPods will automatically update anywhere from a couple of days to a week after the latest monthly release is deployed. This is because PikaPods tests each new release before deploying it.
+For web clients, PikaPods will automatically update about a week or so after the latest monthly release is deployed. PikaPods tests each new release before deploying it. If you have the Notification setting in Actual checked, you will get a popup in Actual notifying you that the server has been updated.
```

Comment:
I don't think this is true, the notification setting in-app refers to the new release notification which is triggered when a new GitHub release is made public, not when Pikapods actually updates.

The "update now" button I believe you're referring to is always active, regardless of that setting.